site/content/en/docs/administration/advanced/webhooks.md

```diff
@@ -64,7 +64,7 @@ To create a webhook for **Organization**, do the following:
 2. Go to the **Organization** > **Settings** > **Actions** > **Setup Webhooks**.
 3. In the top right corner click **+**
 
-  ![](/images/create_organization_webhook.gif)
+  ![Create Organization Webhook](/images/create_organization_webhook.gif)
```

Comment:
IMO, a lot of these alt texts are too terse to be useful. "Create Organization Webhook" doesn't describe the image in a meaningful way. I would expect to see something like "Screen recording of a user navigating to the "Setup a webhook" page for an organization."